setup.py

```diff
@@ -71,9 +73,11 @@ def run(self):
         'git://github.com/NeuralEnsemble/libNeuroML.git#egg=libNeuroML',
         'git://github.com/zopefoundation/ZODB.git#egg=ZODB',
         'git://github.com/RDFLib/FuXi#egg=FuXi-1.4.dev-r0',
+        'git+https://github.com/RDFLib/rdflib-zodb#egg=rdflib-zodb-1.1-dev',
+        'git+https://github.com/mwatts15/YAROM@dev#egg=yarom-0.7.1',
     ],
     version = '0.5.3',
-    packages = ['PyOpenWorm'],
+    packages = ['PyOpenWorm', 'PyOpenWorm.v0', 'PyOpenWorm.graphObjectAdapter'],
```

Comment:
Do we need to keep parallel versions of things around in a 'v0' package?  Why not just make a clean break and consolidate everything we need in the current directory?  Why can't the graphObjectAdapter be in the main PyOpenWorm package?